tests/integration/master/test_clear_funcs.py

```diff
@@ -150,8 +150,9 @@ def test_clearfuncs_config(self):
         }
         ret = clear_channel.send(msg, timeout=5)
         assert not os.path.exists(
-            self.evil_file_path
```

Comment:
This line should be as it is on the master branch.
Dunno why it only failed on windows ...

Reply:
Yeah, that fact that it didn't fail on windows is raising some concerns for me.

Reply:
It failed on windows but not on Linux.

Reply:
Oh, you edited :)

Reply:
Yes, concerning